server/lib/nl/config_builder/main.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+from server.config.subject_page_pb2 import SubjectPageConfig
+from server.lib.nl.common import variable
+from server.lib.nl.common.utterance import ChartType
+from server.lib.nl.common.utterance import Utterance
+from server.lib.nl.config_builder import bar
+from server.lib.nl.config_builder import base
+from server.lib.nl.config_builder import event
+from server.lib.nl.config_builder import map
+from server.lib.nl.config_builder import ranking
+from server.lib.nl.config_builder import scatter
+from server.lib.nl.config_builder import timeline
+
+
+#
+# Given an Utterance, build the final Chart config proto.
+#
+def build(uttr: Utterance,
```

Comment:
nit: might just be me but perhaps we can name this file something other than `main.py`? When I see `main.py` I sort of expect to see a main() function but that's clearly not the intent here.